elements/rh-accordion/rh-accordion-header.css

```diff
@@ -0,0 +1,113 @@
+:host {
+  --_padding-block-start: var(--rh-space-lg, 16px);
+  --_padding-inline-end: var(--rh-space-xl, 24px);
+  --_padding-block-end: var(--rh-space-lg, 16px);
+  --_padding-inline-start: var(--rh-space-xl, 24px);
+  --_text-color: var(--rh-color-text-primary-on-light, #151515);
+  --_active-text-color: var(--rh-color-text-primary-on-light, #151515);
+  --_background-color: #ffffff;
+  --_active-background-color: var(--_rhds-background-color, #f0f0f0);
+  --_font-size: var(--rh-font-size-body-text-md, 1rem);
+  --_after-background-color: transparent;
+  --_expanded-background-color: var(--rh-color-text-brand-on-light, #ee0000);
+}
+
+#heading {
+  color: var(--rh-color-text-primary-on-light, #151515);
+  background-color: var(--_rhds-background-color, var(--rh-color-surface-lightest, #ffffff));
+  font-weight: var(--rh-font-weight-heading-medium, 500);
+}
+
+.dark {
+  --_text-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_background-color: var(--rh-color-surface-darkest, #151515);
+  --_active-background-color: var(--rh-color-surface-darkest, #151515);
+  --_active-text-color: var(--rh-color-text-primary-on-dark, #ffffff);
+  --_expanded-background-color: var(--rh-color-brand-red-on-dark, #ff3333);
+  --_border-right-color: var(--rh-color-black-600, #6a6e73);
+}
+
+:host([large]) {
+  --_font-size: var(--rh-font-size-body-text-lg, 1.125rem);
+  --_padding-block-start: var(--rh-space-lg, 16px);
+  --_padding-inline-end: var(--rh-space-xl, 24px);
+  --_padding-block-end: var(--rh-space-lg, 16px);
+  --_padding-inline-start: var(--rh-space-xl, 24px);
+}
+
+:host([expanded]) {
+  border-right: 2px solid var(--_border-right-color, #d2d2d2);
```

Comment:
I wonder if this should be `border-inline-end` instead of `border-right`.

When the accordion is presented under `dir="rtl"` should the borders also adjust sides?  I noticed that what looks like a brand red border when expanded isn't actually a border `--_panel-content-body-accent-color` so might be a bit more tricky to just switching to logical properties.

Reply:
Updated these with logical properties